middleware/downloadcsv.js

```diff
@@ -8,10 +8,10 @@ const get_qcode = (row) => {
     return row.answers.options.qCode
   }
   if (row.answers.q_code) {
-    return row.answers.q_code
+    return row.answers.secondaryQCode ? `${row.answers.q_code} / ${row.answers.secondaryQCode}` : row.answers.q_code
```

Comment:
This is not needed. The runner version of the date range splits it into 2 answers and uses just q_code not secondary qcode.

Reply:
Have reverted that line